ivy/functional/frontends/paddle/search.py

```diff
@@ -33,6 +33,32 @@ def argsort(x, /, *, axis=-1, descending=False, name=None):
     return ivy.argsort(x, axis=axis, descending=descending)
 
 
+# kthvalue
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+)
+@to_ivy_arrays_and_back
+def kthvalue(x, k, axis=-1, keepdim=False, *, out=None):
```

Comment:
paddle.kthvalue does not have an `out` argument, just a `name`, default value for axis is `None`

Reply:
For the `axis` argument I found in the docs that if `axis` is `None` then the default value of `axis` will be -1, that's why I directly assigned the default value to be -1, but you are CORRECT I should follow the signature anyway sorry for that. And for the `out` yeah my bad :(.